### ISSUE TITLE
Windows re-installs another driver

### ISSUE THREAD

**Issue by Juanitocancino** [xerpi/vitastick#17]
Hello, i have a little problem. But i do not know if my computer is the responsible for  this error ... because the plugin in my PS Vita works fine, the screen turns black and if i wake up my PS vita it says "SELECT + STAR" to exit. but the problem comes when i plug the USB cable, the first try, it set up a new device "PS Vita", like a joystick, but a few seconds later i heard the sound that windows makes when you disconnect something. And now install another device. This time is the "Playstation Vita" the original driver of the PS vita for the PC. What can i do to solve this?

**Comment by xerpi** [comment]
Does it stop working after it installs the new driver?

**Comment by Juanitocancino** [comment]
No... It still working. But if i unplugged and plug in again it says "Playstation Vita" (Official Driver) instead of "Ps Vita"

**Comment by Juanitocancino** [comment]
Hey, I have a little Question... The VPK and the Skprx have to be in the ux0? 
Because i have the Plugin in the ur0 and the vpk in mi SD2Vita
